django/utils/translation/trans_real.py

```diff
@@ -529,7 +529,11 @@ def blankout(src, char):
 
 
 context_re = re.compile(r"""^\s+.*context\s+((?:"[^"]*?")|(?:'[^']*?'))\s*""")
-inline_re = re.compile(r"""^\s*trans\s+((?:"[^"]*?")|(?:'[^']*?'))(\s+.*context\s+((?:"[^"]*?")|(?:'[^']*?')))?\s*""")
+inline_re = re.compile(
+    r"""^\s*trans\s+((?:"[^"]*?")|(?:'[^']*?'))"""
```

Comment:
If you could add any comments to the regex to help a casual reader looking at it, that could be a nice win.